test/setup/platforms/openshift4.ts

```diff
@@ -32,7 +32,7 @@ export async function setupTester(): Promise<void> {
     OPENSHIFT_CLI_VERSION,
   );
   console.log('Downloading OpenShift CLI...');
-  const response = await needle('get', downloadUrl);
+  const response = await needle('get', downloadUrl, { follow_max: 5 });
```

Comment:
so this is to allow the request to be redirected  5 times?

Reply:
Tbh, it's a random number. I've took it from Snyk internal http library